server/src/test/java/org/elasticsearch/action/ActionModuleTests.java

```diff
@@ -234,4 +228,11 @@ public void testCustomRestWrapperDeprecationMessage() {
         public void handleRequest(RestRequest request, RestChannel channel, NodeClient client) throws Exception {
         }
     }
+
+    class SecPlugin implements ActionPlugin {
```

Comment:
Nit: I don't think the abbreviation helps here - why `Sec` rather than `Security` ?